src/models/Tutor.ts

```diff
@@ -1,7 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
```

Comment:
@bigLucas 
For a reason I don't know, is indicating some new errors about modules importations.
At the moment, I am investigating these errors, so far nothing relevant.

**Edit:** Restarting the branch the error over TypeORM came out.


> 1. Error over `PrimaryGeneratedColumn ` :

> Module '"typeorm"' has no exported member 'PrimaryGeneratedColumn'.

![image](https://user-images.githubusercontent.com/68197280/120122404-e8296880-c17e-11eb-9351-15596ccd8c10.png)
.

> 2. Error over `aws-lambda` :

> File 'c:/Users/vntarto/Documents/Mentoring-BranchTypeORM/serverlessoffline/node_modules/@types/aws-lambda/index.d.ts' is not a module.

![image](https://user-images.githubusercontent.com/68197280/120122410-ec558600-c17e-11eb-9cdf-bedccfd5f3f2.png)

Reply:
@Aristevao I don't know what you did, but I don't have these errors, maybe you can reload your vs-code...
Or, you can run `npm install`.

Reply:
@bigLucas
I ran `npm install`, but the error continued. Updating the aws types worked well, but maybe it really could be solved by reloading vs-code. I'll try  next time.